Products.json

```diff
@@ -0,0 +1,121 @@
+[{
```

Comment:
Don't know if you meant to commit this file. The default data for products lives here: https://github.com/reactioncommerce/reaction/blob/master/private/data/Products.json . (if you need to make modifications)